sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * Add optional input argument `connection_name` to methods `.inference.get_chat_completions_client`,
  `.inference.get_embeddings_client` and `.inference.get_azure_openai_client`.
+* Fix for streaming issue, https://github.com/Azure/azure-sdk-for-python/issues/38918
```

Comment:
Should this be under the bug fix section?

Reply:
Also please remove "Breaking Changes" section. And perhaps move the "Bug Fixed to the top".

Reply:
Updated information, thanks